src/elements/InputField/InputField.styl

```diff
@@ -129,7 +129,10 @@
 
   cursor var(--input-field-cursor)
 
-  width 20em
+  height $height-control
+  width $width-control
+  box-sizing $box-sizing-control
+  
 
```

Comment:
Two successive newlines are not allowed.

Reply:
```suggestion
border-top-width var(--input-field-top-border-width)

border-right-width var(--input-field-right-border-width)
```